Makefile

```diff
@@ -31,6 +31,7 @@ pip: pip_build pip_test call_pip_local
 # update current conda environment
 conda_update:
 	conda env update -n $(CONDA_DEFAULT_ENV) --file environment.yml
+	conda env update -n $(CONDA_DEFAULT_ENV) --file hi-ml/testhiml/testhiml/utils/image_loading/environment.yml
 	conda env update -n $(CONDA_DEFAULT_ENV) --file hi-ml/testhiml/testhiml/utils/slide_image_loading/environment.yml
```

Comment:
can we keep those two experimental things out of the conda env please? whoever touches these evaluation scripts needs to do due diligence to set up their own conda env.

Reply:
They are there to make pyright happy. I can change the makefile so that pyright uses a different conda env